content/posts/android/Android Tips.md

```diff
@@ -148,7 +148,7 @@ Depending on the carrier, you may need to enable additional network types for Wi
 
 ## Carrier Tracking
 
-Carriers can track your coarse location via cell towers using the IMSI and IMEI broadcasted by your baseband modem. In order to avoid this type of tracking, you have to enable the airplane mode which would disable the baseband modem.
+Carriers can track your coarse location via cell towers using the IMSI and IMEI broadcasted by your baseband modem. In order to avoid this type of tracking, you have to enable the airplane mode which would disable the baseband modem. In addition, you will also need to disable Wifi calling to avoid IP based tracking by the carrier, as its connection will not go through your VPN tunnel.
```

Comment:
```suggestion
Carriers can track your coarse location via cell towers using the IMSI and IMEI broadcasted by your baseband modem. In order to avoid this type of tracking, you have to enable the airplane mode which would disable the baseband modem. In addition, you will also need to disable Wi-Fi calling to avoid IP based tracking by the carrier, as its connection will not go through your VPN tunnel.
```

Reply:
Not gonna commit this unless the same change is made to every instance of wifi being mentioned lol

Reply:
It's just the correct spelling smh